realm/realm-annotations-processor/src/test/resources/some/test/AllTypes.java

```diff
@@ -80,6 +81,23 @@
     private RealmList<UUID> columnUUIDList;
     private RealmList<Mixed> columnMixedList;
 
+    private RealmDictionary<AllTypes> columnRealmDictionary;
+
+    private RealmDictionary<Boolean> columnBooleanDictionary;
+    private RealmDictionary<String> columnStringDictionary;
+    private RealmDictionary<Integer> columnIntegerDictionary;
+    private RealmDictionary<Float> columnFloatDictionary;
+    private RealmDictionary<Long> columnLongDictionary;
+    private RealmDictionary<Short> columnShortDictionary;
+    private RealmDictionary<Double> columnDoubleDictionary;
+    private RealmDictionary<Byte> columnByteDictionary;
+    private RealmDictionary<byte[]> columnBinaryDictionary;
+    private RealmDictionary<Date> columnDateDictionary;
+    private RealmDictionary<ObjectId> columnObjectIdDictionary;
+    private RealmDictionary<UUID> columnUUIDDictionary;
+    private RealmDictionary<Decimal128> columnDecimal128Dictionary;
+    private RealmDictionary<Mixed> columnMixedDictionary;
```

Comment:
Shouldn't we have a field for RealmDictionary<[embedded object]>?

Reply:
I'm testing that separately in `DictionaryMiscTests`. I decided not to add to `AllTypes` because there are no embedded object fields in the class.